thundra/wrappers/web_wrapper_utils.py

```diff
@@ -72,12 +75,16 @@ def update_application_info(application_info_provider, application_info, app_cla
     })
 
 
+def process_request_route(execution_context, request_route_path):
+    if request_route_path and execution_context and execution_context.scope:
```

Comment:
I just checked Node.js impl and seems that we prefixed `trigger_operation_name` and `TRIGGER_OPERATION_NAMES` tag with hostname.
```nodejs
function handleRoutePath(context: ExecutionContext, resourceName: string) {
    const { rootSpan, request, response } = context;

    const triggerOperationName = request.hostname + resourceName;
    context.triggerOperationName = triggerOperationName;
    context.applicationResourceName = resourceName;

    // Change root span name and response header
    rootSpan.operationName = resourceName;
    response.set(TriggerHeaderTags.RESOURCE_NAME, triggerOperationName);
    InvocationSupport.setAgentTag(SpanTags.TRIGGER_OPERATION_NAMES, [triggerOperationName]);
}
```
I think, it is better to apply same approach here